protocol/app/app.go

```diff
@@ -602,14 +603,14 @@ func New(
 			app.Server.ExpectLiquidationsDaemon(
 				daemonservertypes.MaximumAcceptableUpdateDelay(daemonFlags.Liquidation.LoopDelayMs),
 			)
+			app.LiquidationsClient = liquidationclient.NewClient(logger)
 			go func() {
-				if err := liquidationclient.Start(
```

Comment:
nit: in a separate PR, can we be consistent with `StartNewClient` ([here](https://github.com/dydxprotocol/v4-chain/pull/738/files#diff-7148577e1c863205cc155a436a9c9b806e655073d37a5124da0af8b5e9370317L627)) vs `.*Client.Start`? 

I think we should consistently use `.*Client.Start`

Reply:
https://linear.app/dydx/issue/CORE-756/refactor-pricefeed-client-to-start-in-a-consistent-pattern-with-other